cfme/tests/physical_infrastructure/ui/test_physical_server_list_buttons.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.common.physical_server_views import PhysicalServersView
+from cfme.physical.provider.lenovo import LenovoProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.wait import wait_for
+
+pytestmark = [pytest.mark.tier(3), pytest.mark.provider([LenovoProvider], scope="module")]
+
+
+@pytest.fixture(scope="function")
+def physical_servers(appliance, provider, setup_provider):
+    # Get and return all the physical servers
+    return appliance.collections.physical_servers.all(provider)
+
+
+@pytest.fixture(scope="function")
+def physical_servers_collection(appliance):
+    # Get and return collection of the physical servers
+    return appliance.collections.physical_servers
+
+
+# Configuration Button
+def test_refresh_relationships(physical_servers_collection, physical_servers, provider):
+    view = provider.create_view(PhysicalServersView)
+    last_refresh = provider.last_refresh_date()
+    option = "Refresh Relationships and Power States"
+    physical_servers_collection.custom_button_action("Configuration", option, physical_servers)
+    out, time = wait_for(
+        lambda: last_refresh != provider.last_refresh_date(),
+        fail_func=view.browser.refresh,
+        message="Wait for the servers to be refreshed...",
+        num_sec=300,
+        delay=5
+    )
+    assert out
+
+
+actions = [
+    ("Power", "Power Off", "power_off"),
```

Comment:
**Optional**

In the future, consider using namedtuple for things like this, so they can be referred to be attribute names instead of index locations - this becomes useful when parametrizing a test case, as on L56 below. You can then do something like `ids=[action.id for action in actions]` where each action is a namedtuple.

Reply:
I'm using this suggestion now , thank you.